internal/common/common.go

```diff
@@ -17,6 +17,8 @@ import (
 
 const MinMasterHostsNeededForInstallation = 3
 
+const ServiceCACert = "/etc/assisted-service/service-ca-cert.crt"
```

Comment:
Also, is this the best place for a constant like this? It feels wrong here. Not sure where else it should go though. Maybe @eranco74 can suggest something?

Reply:
That's a good point and it did confuse me a bit. I will rename those to ServiceCACertPath and HostCACertPath.

> this PR is configuring the CA cert for the discovery ignition, controller pod, and the master/worker ignition (I still don't know a better name for this one), right? 
Yes, that's right. This PR doesn't configure SSL on the api-service itself. In the onprem usecase, we are fronting the api-service with nginx with ssl. But assuming the service is behind SSL, this configures the discovery ignition, master/worker ignition and the controller with CA Cert so they can talk to the service over SSL.